TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -777,6 +782,7 @@ void TTSSpeaker::GStreamerBusWatchThreadFunc(void *ctx) {
                 speaker->handleMessage(msg);
                 gst_message_unref(msg);
```

Comment:
"TTSSpeaker::GstBusCallback" is not used anymore and hence removed

Reply:
Modified the code.